src/Relation/Binary/Properties/PartialSetoid.agda

```diff
@@ -0,0 +1,46 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Additional properties for setoids
+------------------------------------------------------------------------
+
+{-# OPTIONS --cubical-compatible --safe #-}
+
+open import Relation.Binary.Bundles using (PartialSetoid)
+
+module Relation.Binary.Properties.PartialSetoid
+  {a ℓ} (S : PartialSetoid a ℓ) where
+
+open import Data.Product.Base using (_,_; _×_)
+open import Relation.Binary.PropositionalEquality.Core as ≡ using (_≡_)
+
+open PartialSetoid S
+
+private
+  variable
+    x y z : Carrier
+
+------------------------------------------------------------------------
+-- Proofs for partial equivalence relations
+
+trans-reflˡ : x ≡ y → y ≈ z → x ≈ z
+trans-reflˡ ≡.refl p = p
+
+trans-reflʳ : x ≈ y → y ≡ z → x ≈ z
+trans-reflʳ p ≡.refl = p
+
+p-reflˡ : x ≈ y → x ≈ x
```

Comment:
What does `p` stand for here?

Reply:
see above! under "issues: naming" ;-)

Reply:
My vote would be for expanding these out to `partial`, as I don't think they are guessable from the `p` alone?

Reply:
Happy to do so!

Reply:
I think `partial-` is better than `p-`. I do like to import a variety of things qualified, so I would personally drop the `partila-` completely, but I'm definitely not going to insist on that.